docs/developer-guide.md

```diff
@@ -8,7 +8,7 @@ In this guide we will walk through how to build applications on top of Styx.
 
 ## 1.2 System Requirements
 
-Building Styx requires Java 11.
+Building Styx requires Java 21.
 
 The build system requires Apache Maven. The Styx team uses Maven version 3.2.1
```

Comment:
maybe just generic latest version of maven 3?

Reply:
Changed it in #848